### ISSUE TITLE
sqlValidateFK has SQL syntax error

### ISSUE THREAD

**Issue by jonathanjouty** [scrive/hpqtypes-extras#27]
```hs
-- | Validate foreign key previously created as NOT VALID.
sqlValidateFK :: RawSQL () -> ForeignKey -> RawSQL ()
sqlValidateFK tname fk = "VALIDATE" <+> fkName tname fk
```

Should be `VALIDATE CONSTRAINT $fkName`
See: https://www.postgresql.org/docs/current/sql-altertable.html

**Comment by jonathanjouty** [comment]
```
DBException {dbeQueryContext = RawSQL "ALTER TABLE table_name VALIDATE fk__table_name__column_name__foreign_table_name" (), dbeError = DetailedQueryError {qeSeverity = "ERROR", qeErrorCode = SyntaxError, qeMessagePrimary = "syntax error at or near \"fk__table_name__column_name__foreign_table_name\"", qeMessageDetail = Nothing, qeMessageHint = Nothing, qeStatementPosition = Just 39, qeInternalPosition = Nothing, qeInternalQuery = Nothing, qeContext = Nothing, qeSourceFile = Just "scan.l", qeSourceLine = Just 1134, qeSourceFunction = Just "scanner_yyerror"}}
```

**Comment by arybczak** [comment]
Fixed in 37fb5b8696d3c6f37cdc7b2e9e94f630b3934fea..
